test/e2e-tests.js

```diff
@@ -305,8 +322,27 @@ describe('matrix-public-archive', () => {
 
       const dom = parseHTML(archivePageHtml);
 
+      // Make sure the user display name is visible on the message
+      assert.match(
+        dom.document.querySelector(`[data-event-id="${imageEventId}"]`).outerHTML,
+        new RegExp(`.*${escapeStringRegexp(displayName)}.*`)
+      );
+
+      // Make sure the user avatar is visible on the message
+      const avatarImageElement = dom.document.querySelector(
+        // FIXME: Use more stable select here instead of `.avatar`,
+        // see https://github.com/vector-im/hydrogen-web/pull/773
+        `[data-event-id="${imageEventId}"] .avatar img`
```

Comment:
Proposed a change in Hydrogen at https://github.com/vector-im/hydrogen-web/pull/773